conf/machine/include/rpi-base.inc

```diff
@@ -102,7 +102,7 @@ SERIAL_CONSOLES_CHECK ??= "${SERIAL_CONSOLES}"
 BOOTFILES_DIR_NAME ?= "bootfiles"
 
 # Set Raspberrypi splash image
-SPLASH = "psplash-raspberrypi"
+SPLASH ?= "psplash-raspberrypi"
```

Comment:
oe-core also has `meta/classes/image.bbclass` `SPLASH ?= "psplash"` have you tested which one gets preferred when external setting is not in play ?

Reply:
Thanks @kraj for the review.

Yes, checked on my side. And when `SPLASH ?= "psplash-raspberrypi"` is like this, so, overridable and without an external setting (e.g `SPLASH = foo`), `psplash-raspberry` is preferred.

Otherwise, for safe version, I can introduce a new global variable `RPI_SPLASH_NAME` or something else  ... (default = `RPI_SPLASH_NAME ?= "psplash-raspberrypi" `) and update the `SPLASH` variable as follow:

`SPLASH = "${RPI_SPLASH_NAME}" `

Let me know.

Reply:
no need for new variable. We want to use core variable as much as possible. If the order is

custom > meta-rpi > oe-core  then I am ok

Reply:
Sure, you're right @kraj, so in this case, this PR is working as expected ;)